src/sign.builds

```diff
@@ -15,6 +15,12 @@
     <WindowsNativeLocation Include="$(BinDir)*.dll" />
     <WindowsNativeLocation Include="$(BinDir)*.exe" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(BuildArch)' == 'x86'">
+    <!-- Sign api-ms-win-core-xstate-l2-1-0 binary as it is only catalog signed in the current SDK. -->
+    <WindowsNativeLocation Include="$(BinDir)Redist\ucrt\DLLs\$(BuildArch)\api-ms-win-core-xstate-l2-1-0.dll" />
```

Comment:
by the way, any idea if the casing matters? It's currently `API-MS-Win-core-xstate-l2-1-0.dll`

Reply:
nope.  It's windows.  This is the same code in 2.1